tests/PHPUnit/System/TwoVisitsWithCustomVariablesSegmentMatchVisitorTypeTest.php

```diff
@@ -84,22 +86,21 @@ public function testCheck()
             //   )
             'archive_blob_2010_01'    => 20,
             // This contains all 'last N' weeks & days,
-            // (2 metrics
+            // (6 metrics
             //  + 2 referrer metrics
             //  + 3 done flag )
             //  * 2 segments
-            // + 1 Done flag per Plugin, for each "Last N" date
-            'archive_numeric_2010_01' => 138,
+            // for each "Last N" date that has data (just one date)
+            'archive_numeric_2010_01' => 22,
```

Comment:
just double checking @diosmosis do you maybe know why it were before so many entries? is it less now because we delete some archives?

Reply:
This PR has a big optimization for sites/periods that have no visits (see PR description). If we see there will be no visits for an archive, we don't even initiate archiving, so no archive is created.